eth2prysm/templates/beacon-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.beacon.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.beacon.label }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.beacon.label }}
+  replicas: 1
+  strategy: 
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.beacon.label }}
+    spec:
+      containers:
+      - name: {{ .Values.beacon.name }}
+        image: gcr.io/prysmaticlabs/prysm/beacon-chain:{{ .Values.imageVersion }}
+        args:
+        - --{{ .Values.ethereumNetwork }} 

```

Comment:
is this optional?

Reply:
Since prysm doesn't require "mainnet" flag, it's technically optional. I changed it to a optional ethereumTestnet flag to avoid unnecessary attention. If users want to run on testnet, they can specify the flag. Otherwise, it'll default to mainnet just like what prysm client does.